lib/new_relic/distributed_trace.ex

```diff
@@ -112,7 +112,10 @@ defmodule NewRelic.DistributedTrace do
   end
 
   def report_attributes(
-        %Context{parent_id: nil} = context,
+        %Context{
+          parent_id: nil,
+          span_guid: nil
+        } = context,
```

Comment:
Only when there's no "id" AND no "tx" on the incoming payload do we consider there to be no parent, otherwise we do and the `report_attributes` function head below is executed

Reply:
So, if either of them is nil, the other `report_attributes` function is used, and either `parentId` or `parentSpanId` is set to nil.  And then, somewhere else, the nil attributes will be excluded from the event sent to the back end, right?

Reply:
Right, if _both_ are nil, this is the function head that gets picked, otherwise the one below. Then later on all `nil` values get dropped like the spec says they should have been all along :)